docs/build/apps/wallet/sep7.mdx

```diff
@@ -0,0 +1,134 @@
+
+## Tx Operation
+
+The tx operation represents a request to sign a specific transaction envelope, with [some configurable parameters].
+
+<CodeExample>
+
+```typescript
+const sourceAccountKeyPair = "G...";
+const destinationAccountKeyPair = "G...";
+const txBuilder = await stellar.transaction({
+  sourceAddress: sourceAccountKeyPair,
+});
+const tx = txBuilder.createAccount(destinationAccountKeyPair).build();
+const xdr = encodeURIComponent(tx.toEnvelope().toXDR().toString("base64"));
+const callback = encodeURIComponent("https://example.com/callback");
+const txUri = `web+stellar:tx?xdr=${xdr}&callback=${callback}`;
+const uri = wallet.parseSep7Uri(txUri);
+// uri can be parsed and transaction can be signed/submitted by an application that implements Sep-7
+```
+
+</CodeExample>
+
+You can set replacements to be made in the xdr for specific fields by the application, these will be added in the [Sep-11 transaction representation format] to the URI.
+
+<CodeExample>
+
+```typescript
+const uri = new Sep7Tx(txUri);
+uri.addReplacement({
+  id: "X",
+  path: "sourceAccount",
+  hint: "account from where you want to pay fees",
+});
+```
+
+</CodeExample>
+
+You can assign parameters after creating the initial instance using the appropriate setter for the parameter.
+
+<CodeExample>
+
+```typescript
+const sourceAccountKeyPair = "G...";
+const destinationAccountKeyPair = "G...";
+const txBuilder = await stellar.transaction({
+  sourceAddress: sourceAccountKeyPair,
+});
+const tx = txBuilder.createAccount(destinationAccountKeyPair).build();
+const uri = wallet.Sep7Tx.forTransaction(tx);
+uri.callback = "https://example.com/callback";
+uri.msg = "here goes a message";
+uri.toString(); // encodes everything and converts to a uri string
+```
+
+</CodeExample>
+
+## Pay Operation
+
+The pay operation represents a request to pay a specific address with a specific asset, regardless of the source asset used by the payer. You can [configure parameters] to build the payment operation.
+
+<CodeExample>
+
+```typescript
+const destination = "G...";
+const assetIssuer = "G...";
+const assetCode = "XLM";
+const amount = "120.1234567";
+const memo = "memo";
+const message = encodeURIComponent("pay me with lumens");
+const originDomain = "example.com";
+const payUri = `web+stellar:pay?destination=${destination}&amount=${amount}&memo=${memo}&msg=${message}&origin_domain=${originDomain}&asset_issuer=${assetIssuer}&asset_code=${assetCode}`;
+const uri = parseSep7Uri(payUri);
+// uri can be parsed and transaction can be built/signed/submitted by an application that implements Sep-7
+```
+
+</CodeExample>
+
+You can assign parameters after creating the initial instance using thr appropriate setter for the parameter.
```

Comment:
```suggestion
You can assign parameters after creating the initial instance using the appropriate setter for the parameter.
```

Reply:
thanks for catching, fixed in ff3c968ffdca003c792112ead948897e000d68e7